src/util/rangeutil.hpp

```diff
@@ -194,3 +195,15 @@ Value max_value(const Seq& seq, Compare cmp = Compare{}) {
 } // namespace util
 } // namespace mc
 } // namespace nest
+
+// convenience helper for printing a range
+
+template <typename T>
+std::ostream& operator<<(std::ostream& o, const nest::mc::util::range<T>& rng) {
+    o << "{";
+    for (auto const& value: rng) {
+        o << value << " ";
+    }
+    return o << "}";
+}
```

Comment:
I see your point. That Procrustes sounds like quite an unsavoury character, so I am sorry if my one-size-fits-all helper function caused harm to any travellers crossing our code base.